lib/kafka_ex/new/client.ex

```diff
@@ -22,7 +22,9 @@ defmodule KafkaEx.New.Client do
   use GenServer
 
   @type args :: [
-          KafkaEx.worker_setting() | {:allow_auto_topic_creation, boolean}
+          KafkaEx.worker_setting()
+          | {:allow_auto_topic_creation, boolean}
+          | {:iniital_topics, [binary]}
```

Comment:
iniital -> initial

Reply:
Fixed.  I had to actually move this to `KafkaEx.worker_settings` to get dialyzer to pass.